eclair-core/src/main/resources/reference.conf

```diff
@@ -163,12 +163,20 @@ eclair {
 
     // number of blocks to target when computing fees for each transaction type
     target-blocks {
-      funding = 6                       // target for the funding transaction
-      commitment = 2                    // target for the commitment transaction (used in force-close scenario) *do not change this unless you know what you are doing*
-      commitment-without-htlcs = 12     // target for the commitment transaction when we have no htlcs to claim (used in force-close scenario) *do not change this unless you know what you are doing*
-      mutual-close = 12                 // target for the mutual close transaction
-      claim-main = 12                   // target for the claim main transaction (tx that spends main channel output back to wallet)
-      safe-utxos-threshold = 10         // when our utxos count is below this threshold, we will use more aggressive confirmation targets in force-close scenarios
+      // target for the funding transaction
+      funding = 6
+      // target for the commitment transaction (used in force-close scenario) *do not change this unless you know what you are doing*
+      commitment = 2
+      // target for the commitment transaction when we have no htlcs to claim (used in force-close scenario) *do not change this unless you know what you are doing*
+      commitment-without-htlcs = 12
+      // if false, the commitment transaction will not be fee-bumped when we have no htlcs to claim (used in force-close scenario) *do not change this unless you know what you are doing*
+      spend-anchor-without-htlcs = true
```

Comment:
Nit: I would have put that in the upper section, so that `target-blocks` stays homogeneous.

Reply:
That makes more sense indeed, done in https://github.com/ACINQ/eclair/pull/2246/commits/82e1f5e01819353dd88114d16bfd32d6f6e03398